### ISSUE TITLE
Can't add downloader sources

### ISSUE THREAD

**Issue by fncll** [mrichards42/xword#197]
No matter what I do, when I hit OK after creating a new downloader source, nothing happens. No new entry appears in the list. It's as if I am canceling the operation, but I am not!

**Comment by mrichards42** [comment]
Yes, apologies the downloader config is kind of a mess right now, and I'm thinking about simplifying how sources are configured so a lot of that will change when I make the time for it.

In the mean time if you're up for hand-editing some files, you can:

1. [Make sure the config directory actually exists](https://github.com/mrichards42/xword/issues/116#issuecomment-656101422) -- or if you want to grab a recent unreleased build, this problem is fixed (builds are https://ci.appveyor.com/project/MikeRichards/xword/history, click on a build number, find the appropriate job, and look at the artifacts tab)

**Comment by ilikepeaches** [comment]
Unfortunately, appveyor says, "Artifacts older than 1 month are automatically deleted."
